### ISSUE TITLE
Circular Progressbar not working

### ISSUE THREAD

**Issue by Mrxx99** [AvaloniaCommunity/Material.Avalonia#195]
Since nuget package 3.0.0-avalonia11-preview2.119-nightly and up, setting the class "circular" on a ProgressBar does nothing and it still is displayed linear. Since it works in the demo app on latest dev I guess it is fixed there. Could you please release a new nightly package?

I can't downgrade because I am using Snackbar with buttons which is not supported in earlier versions.
I know nightlys are not stable but I am currently porting a WPF app and need that features and for now can live with nightly packages.

**Comment by SKProCH** [comment]
Hhm, this is kinda strange. Demo has the latest version of nightly package already (since the nightly was build on top of each commit to dev/3.0 branch)

**Comment by SKProCH** [comment]
Hello, @Mrxx99 

Seems like you need to add 
```xaml
<StyleInclude Source="avares://Material.Styles/Resources/Compatibility/Index.axaml"/>
```
to your `Styles` section in App.axaml file. 

May be we make it included by default in future.

**Comment by SKProCH** [comment]
Also, implemented by default in `3.0.0-avalonia11-preview2.124-nightly`
